.github/workflows/pull-request.yml

```
@@ -187,6 +187,58 @@
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Get changed files
+        id: changed-files
+        uses: tj-actions/changed-files@v45
+
+      - name: Identify nearest changed directories
+        id: nearest_dirs
+        run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:11:23: Double quote to prevent globbing and word splitting

(shellcheck)